### ISSUE TITLE
ViewPropTypes will be removed from React Native. Migrate to ViewPropTypes exported from 'deprecated-react-native-prop-types

### ISSUE THREAD

**Issue by Himatheja** [leecade/react-native-swiper#1352]
Package Update Request

### Version
Which versions are you using:

- react-native-swiper v1.6.0
- react-native v0.70.1

Can we have the package supporting the react native versions 0.68 and above as ViewPropTypes is deprectaed?

Right now, I'm using patch package to fix this issue.

**Comment by Himatheja** [comment]
Since ViewPropTypes is declared and is being not used. We can remove the declaration in src/index.js

**Comment by Lakston** [comment]
is there a patch planned for this ?

**Comment by mrtwebdesign** [comment]
I don't think this is actively supported anymore. I'm using patch-package to keep it working until I can find a replacement.

**Comment by Lakston** [comment]
That's what I did too, if anyone has a nice replacement for this lib please post it here :)
